### DISCUSSION TITLE
Image upload in Project's sidebar doesn't support high-res (Retina, 2x) images

### DISCUSSION THREAD

**Discussion by borekb** [community/community#19965] [Projects and Issues]
This is the beautiful little UX thing **in an issue** when trying to upload a 2x high resolution image (e.g., a screenshot taken on a Retina display):

<img width="450" alt="Screen Shot 2022-07-04 at 10 45 50" src="https://user-images.githubusercontent.com/101152/177118497-f0b557ea-7507-42bf-83e7-3efae4e99fd5.png">

<img width="450" alt="Screen Shot 2022-07-04 at 10 45 56" src="https://user-images.githubusercontent.com/101152/177118511-3532a28f-7b01-4c6b-aa48-70bbc225e9af.png">

Note how the text area converted the `![image]()` markdown to `<img width="" src="">` for correct rendering.

---

The new **projects sidebar** doesn't do that:

<img width="450" alt="Screen Shot 2022-07-04 at 10 44 41" src="https://user-images.githubusercontent.com/101152/177118838-1d80a649-f76a-4a8d-ac57-63344c5ff450.png">

<img width="450" alt="Screen Shot 2022-07-04 at 10 44 49" src="https://user-images.githubusercontent.com/101152/177118849-136a1c30-4f10-4c3f-8cc7-19463852df7f.png">

Even from the placeholder text, it seems like it's a completely different implementation of the feature which is a pity – `<img>` is "correct" for Retina screenshots, plain `![image]()` syntax will make the image look ugly (twice as large).

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by borekb** [reply]
This feels more like a difference of implementation, not a straight bug, and I believe that these discussions are meant to gather feedback like that.

**Comment by diegoleondev** [comment]
Github is not an image repository, optimize your content for the platform.

**Reply by borekb** [reply]
Sorry but this is not what this feedback is about...
